docker-image-src/5/neo4jlabs-plugins.json

```diff
@@ -1,7 +1,12 @@
 {
   "apoc": {
     "location": "/var/lib/neo4j/labs/apoc-*-core.jar",
-    "versions": "https://github.com/neo4j/apoc/blob/master/versions.json",
+    "properties": {
+      "dbms.security.procedures.unrestricted": "apoc.*"
+    }
+  },
```

Comment:
The way the Neo4j container works, is that it will always check the location first and it will always find an APOC jar. Thus, the versions has no effect and should be removed to reduce the chance of confusing future maintainers.